generators/client/needle-api/needle-client-angular.js

```diff
@@ -199,14 +199,24 @@ module.exports = class extends needleClientBase {
         this.addIcon(iconName);
     }
 
+    _addRoute(route, modulePath, moduleName, needleName, filePath, prefix = ',') {
+        const isRouteAlreadyAdded = jhipsterUtils.checkStringInFile(filePath, `path: '${route}'`, this.generator);
+        if (isRouteAlreadyAdded) {
+            return;
+        }
+        const errorMessage = `${chalk.yellow('Route ') + route + chalk.yellow(` not added to ${filePath}.\n`)}`;
+        const routingEntry =
+            `${prefix}\n` +
```

Comment:
Why don't you put everything inside the same template? Instead of concatenating?
I will be cleaner and easier to read.
It will be formatter by prettier anyway.
Ex:
https://github.com/jhipster/generator-jhipster/blob/dbdac007432ddfb7a2f129b2ce991cd31679211b/generators/client/needle-api/needle-client-angular.js#L174-L179

Reply:
If this is used by some module and this module is not registering prettier transform then module output (and also added route) is not formatted.

Maybe just in case use correct spacing already in generated content?

Reply:
Sure.

Reply:
OK, then I leave this as it is, as such way this is better readable.

Reply:
I think it is more readable using strip margin.
https://github.com/jhipster/generator-jhipster/blob/dbdac007432ddfb7a2f129b2ce991cd31679211b/generators/client/needle-api/needle-client-angular.js#L159-L163

Reply:
Done: now using `stripMargin`.